seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/SeaTunnelServer.java

```diff
@@ -253,4 +257,63 @@ private void printExecutionInfo() {
             coordinatorService.printJobDetailInfo();
         }
     }
+
+    public SeaTunnelConfig getSeaTunnelConfig() {
+        return seaTunnelConfig;
+    }
+
+    public NodeEngineImpl getNodeEngine() {
+        return nodeEngine;
+    }
+
+    public ConnectorPackageService getConnectorPackageService() {
```

Comment:
Why not put it into `CoordinatorService` since connectorPackageService only work on master node.

Reply:
Because in the previous incorrect design, during the initialization of the ConnectorPackageService, the CoordinaterService would be obtained first, and at this time, the CoordinaterService has not yet completed the initialization process. My design here is incorrect. Thank you very much for your reminder. I will put the ConnectorPackageService into the CoordinatorService.